recipe/meta.yaml

```diff
@@ -62,24 +68,22 @@ outputs:
         - llvmdev {{ llvm_version }}
         - make  # [not win]
         - maven
-        - vs2022_win-64  # [win]
         - level-zero-devel  # [not win]
         - pkg-config  # [not win]
         - llvm-spirv {{ llvm_version }}  # [not win]
         - libllvmspirv {{ llvm_version }}  # [not win]
       host:
-        # If no version limit is specified for python then conda goes
-        # crazy and is unable to create host env on windows.
-        - python {{ python }}
+        - python
         - cython
         - numpy
         - tbb-devel
+        - gflags  # [unix]
         - glog
         # Folly and fmt are mutially exclusive on linux but fmt is
         # requred on windows.
         - fmt  # [win]
         - folly >=2022.11.07.00  # [not win]
-        - libsqlite 3.40.0  # [not win]
+        - libsqlite 3.40.0
```

Comment:
How do all think about using >=3.40.0 here? Looks like this is the only place when we have such a strict version requirement

Reply:
this probably comes from the previous version, >= should work as well IMO

Reply:
OK. libsqlite is actually [part](https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/800edfcc759a8f7ac1055290bb72191fa08acc7e/recipe/conda_build_config.yaml#L525-L526) of the global pinning and has a very compatible ABI-policy. So generally relying on that will always pull in the newest libsqlite upon build time (say 3.x) and then generate a dependency on `libsqlite >=3.x,<4` at runtime through a [run-export](https://github.com/conda-forge/sqlite-feedstock/blob/ff23d3eda65af793564a1794250f76864cacc985/recipe/meta.yaml#L39).